README.md

```diff
@@ -17,6 +17,24 @@ Parameters in some Making Tax Digital for Income Tax APIs map to box numbers in
 **Note:** The date shown is the date that the change was released to Sandbox or Production.
 
 ---
+### 25 April 2025 ###
+
+#### Self Assessment Accounts API ####
+
+#### Removed ####
+
+The `INCONSISTENT_QUERY_PARAMS` and `INVALID_DATE_RANGE` Gov-Test-Scenarios are no longer available in the Sandbox for the following endpoint:
```

Comment:
Should avoid single bullet lists, `The `INCONSISTENT_QUERY_PARAMS` and `INVALID_DATE_RANGE` Gov-Test-Scenarios are no longer available in the Sandbox for the 'Retrieve Self Assessment Balance and Transactions' endpoint.

Make the endpoint name a hyperlink to the endpoints page.